### ISSUE TITLE
Hi, I think there's something wrong with your code!

### ISSUE THREAD

**Issue by Erhushenshou** [raytroop/YOLOv3_tf#5]
I have some puzzles with tour code in preprocess_true_boxes part in yolo layer. I wanna contact with you. It seems that you're a Chinese. If so, we can contact through qq. My qq is 815301859. If not, then we shall contact by email. My email address is 815301859@qq.com. I hope to hear from you soon. You are pretty good. I have completed MTCNN and Faster RCNN already. I think we can communicate more. Thank U.

**Comment by Erhushenshou** [comment]
it seems that in your preprocess_true_boxes part, you use tf.reduce_sum to merge the mask and scale into one. But there is an exception when the two ground truth boxes center drop into the same cell and,simultaneously matches the same anchor. If this happens, you have made their value twice.
